### PR TITLE
Added Bat Parrot

### PR DESCRIPTION
**Pull request by david-yieldify**
Added a new guest parrot that is the silent protector and ever watching

### CODE REVIEW

**Comment by cavpollo** [comment]
Bahahaha, I didn't know I needed this, but I love it.

Just a silly optional request. Could it show its mouth, like the original batman where it only covers from its nose up?

**Comment by jmhobbs** [comment]
🙌 Yes!  I like @cavpollo's suggestion too, that'd be neat.

Also, looks like you put the gif into `guests` but the entry into `parrots.json`.  If you can move that over to `guests.json` we will be good to merge 👍

**Comment by david-yieldify** [comment]
Moved as requested! 

Unfortunately I no longer have the original batparrot PSD so will create another batparrot with this style at another date. 
I've the idea of doing different bat suits for different moods.

File: guests.json [hunk 1]
```diff
@@ -76,5 +76,10 @@
   {
     "hd": "hd/partycage.gif",
     "name": "Party Cage"
+  },
+  {
+    "hd": "guests/hd/batparrot.gif",
```
Comment (jmhobbs) [inline guests.json]: Can you make it just `hd/batparrot.gif`?  Sorry!

Comment (david-yieldify) [reply guests.json]: Oh good lord! 
Don't apologise...
🤦

**Comment by jmhobbs** [comment]
Done!  Thanks for contributing!

**Pull request merged**